extlib/benz/include/picrin/compat.h

```diff
@@ -23,6 +23,20 @@ extern "C" {
 # define offsetof(s,m) ((size_t)&(((s *)NULL)->m))
 #endif
 
+#if __STDC_VERSION__ >= 199901L
+# include <stdint.h>
+#else
+# if INT_MAX > 2147483640L      /* borrowed from luaconf.h */
+typedef int int32_t;
+typedef unsigned int uint32_t;
+# elif LONG_MAX > 2147483640L
```

Comment:
ISO C requires that LONG_MAX is at least 2147483647, so this ``elif ...`` can replace with ``else`` unless we are willing to support insane platforms.